### DISCUSSION TITLE
I had deposited 11 dollars through usdc from my wld app but it didn't reflected in my bybit account

### DISCUSSION THREAD

**Discussion by Badri9045** [blockscout/blockscout#11282] [Support]
I had deposited 11 dollars through usdc from my wld app but it didn't reflected in my bybit account

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
